Thrive.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you revert this line

Reply:
This line is the bane of my existence right now

Reply:
Maybe replacing the file with the one in the master branch will work, and then just manually readd ChromaticFilter.cs to the ItemGroup. That might be a bit of a hassle though